pyproject.toml

```diff
@@ -0,0 +1,85 @@
+[build-system]
+requires = ["setuptools>=61.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "djangorestframework"
+readme = "README.md"
+authors = [{name = "Tom Christie", email = "tom@tomchristie.com"}]
+license = {text = "BSD"}
+description = "Web APIs for Django, made easy."
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Environment :: Web Environment",
+    "Framework :: Django",
+    "Framework :: Django :: 4.2",
+    "Framework :: Django :: 5.0",
+    "Framework :: Django :: 5.1",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "Programming Language :: Python :: 3 :: Only",
+    "Topic :: Internet :: WWW/HTTP",
+]
+requires-python = ">=3.8"
+dependencies = [
+    "django>=4.2",
+    'backports.zoneinfo;python_version<"3.9"',
+]
+dynamic = ["version"]
+
+[tool.setuptools.dynamic]
+version = {attr = "rest_framework.__version__"}
+
+[project.urls]
+Homepage = "https://www.django-rest-framework.org/"
+Funding = "https://fund.django-rest-framework.org/topics/funding/"
+Source = "https://github.com/encode/django-rest-framework"
+Changelog = "https://www.django-rest-framework.org/community/release-notes/"
+
+[tool.setuptools]
+zip-safe = false
+include-package-data = true
+license-files = ["LICENSE.md"]
+
+[tool.setuptools.packages.find]
+namespaces = false
```

Comment:
I disabled them because they were disabled originally, using [`find_packages` in the setup.py](https://github.com/encode/django-rest-framework/blob/b25028ac8f46984459f9d9e28069b797986ae476/setup.py#L83) instead of `find_namespace_packages`. I tried your suggestion but it broke the CI, so I reverted it.

Reply:
I see, thanks for trying. It appears I misunderstood the default behavior for nested packages.

You'd need to try the following:

```suggestion
[tool.setuptools.packages.find]
include = ["rest_framework*"]
```

I used the `build` module to build a wheel and with my original suggestion it was warning that the sub-packages were not being included, but with the `include` statement it worked as intended and bundled the sub-packages.

Do you mind running that by the CI?

Reply:
sure. But as it is changing the behavior, even if it works, I would like it in a separate PR :)